mne/preprocessing/bads.py

```diff
@@ -38,3 +38,176 @@ def find_outliers(X, threshold=3.0, max_iter=2):
 
     bad_idx = np.where(my_mask)[0]
     return bad_idx
+
+
+def corrmap(icas, template, 
+            threshold="auto", 
+            name="marked_ics", 
+            verbose_return=False, 
+            plot=True,
+            inplace=False):
```

Comment:
@jona-sassenhagen no reason to use line breaks here
Also note that we use `@verbose` decorater together with a `verbose` keyword.